kolibri/core/tasks/api.py

```diff
@@ -56,14 +56,22 @@ def validate_create_req_data(self, request):
         validated_jobs = []
 
         for request_data in request_data_list:
-            # Make sure the task is registered
+            # Make sure the task is registered.
             registered_task = TaskRegistry.validate_task(request_data.get("type"))
 
+            # Make sure the job's data is valid along with `enqueue_args`.
             job = registered_task.validate_job_data(request.user, request_data)
 
+            # Get validated `enqueue_args`.
+            enqueue_args = registered_task.get_validated_enqueue_args(
```

Comment:
Not sure why this needs to be validated again? It should already have been validated by the job data validation.